build/common.project.props

```diff
@@ -124,7 +124,8 @@
     <MinimumVisualStudioVersion>15.0</MinimumVisualStudioVersion>
     <VsixOutputDirName>VS15</VsixOutputDirName>
     <CodeAnalysisRuleSet Condition="'$(CodeAnalysisRuleSet)' == ''">$(BuildCommonDirectory)NuGet.ruleset</CodeAnalysisRuleSet>
-    <EnforceCodeStyleInBuild Condition=" '$(EnforceCodeStyleInBuild)' != '' " >true</EnforceCodeStyleInBuild>
+    <EnforceCodeStyleInBuild Condition="'$(EnforceCodeStyleInBuild)' == ''">true</EnforceCodeStyleInBuild>
+    <WarningsNotAsErrors>$(WarningsNotAsErrors);EnableGenerateDocumentationFile</WarningsNotAsErrors>
```

Comment:
I am curious to learn more about the benefits of `WarningsNotAsErrors` and `EnableGenerateDocumentationFile` properties.

Reply:
EnableGenerateDocumentationFile is a warning that causes the build to fail when EnforceCodeStyleInBuild is enabled and IDE0005 is set to warning or greater. IDE0005 sets a warning for unused import statements. 

So I'm setting EnableGenerateDocumentationFile to not error on compile.  